src/lib/components/product.svelte

```diff
@@ -5,42 +5,98 @@
   export let image: string
   export let name: string
   export let brand: string
+  export let size: string | undefined
+  export let region: string | undefined
 </script>
 
 <!-- <a href="#"> -->
-<div in:fade={{ duration: 300, delay: index * 150 }} class="product">
+<div in:fade={{ duration: 300, delay: index * 150 }} class="card">
   <img src={image} alt="" />
-  <h2>{brand}</h2>
-  <h4>{name}</h4>
+
+  <div class="content">
+    <h2 class="title">{brand} {name}</h2>
+    <p class="origin"><stronge>Oprindelse</stronge></p>
+    <p class="region">{region}</p>
+    <div class="footer">
+      <p>Spanien</p>
+      <p class="size">{size}</p>
+    </div>
+  </div>
 </div>
 
 <!-- </a> -->
 
 <style>
```

Comment:
Please cleanup styles and remove unused properties and group styles more efficiently, very hard to read & understand